### ISSUE TITLE
Allow imports from Streaks

### ISSUE THREAD

**Issue by hiqua** [iSoron/uhabits#1283]
## Description

Follow-up from: https://github.com/iSoron/uhabits/discussions/1261#discussioncomment-1965306

I think it's this app: https://streaksapp.com/

Sample:
[streaks.csv](https://github.com/iSoron/uhabits/files/7953601/streaks.csv)

**Comment by hiqua** [comment]
@davidskeck could you share a larger sample? The one you shared only has null values for the last two fields, and always the same value for the `entry_type`, can other values appear?

If you used all the features in Streaks, I'm basically interested in the output of:
```
cut -d, -f 4 streaks.csv | sort -u
cut -d, -f 7 streaks.csv | sort -u
cut -d, -f 8 streaks.csv | sort -u
```
if streaks.csv is your full backup.

**Comment by davidskeck** [comment]
Yes, I checked the entirety of my output file and the last two values are zero for me, however, they can be different values (which I just verified in the app).

`page` denotes which page the task appears on, the app has up to 4 pages, 0 indexed.
`quantity` doesn't appear to refer to anything in the tasks I was able to set up. Even tasks that should be completed multiple times didn't update the quantity. I also tried tasks with a specific number goal (Walk the Dog 2x for example), but that number never changed from 0.0.

As for the `entry_type`, it can change, but I don't know what the other values would be. Streaks can automatically mark a task as done by reading data from the number of steps walked for example.

I hope that helps. Thank you!

**Comment by iSoron** [comment]
Not sure there is enough demand for this feature. Are you planning to implement this, @hiqua? If not, I'll move the issue to feature requests, so that we can collect upvotes.

**Comment by hiqua** [comment]
I don't like to commit to this kind of things, so yes let's move this for now.
